src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/tasks/ImportReportsDataTask.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.tasks
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.pecs.jpc.config.TimeSource
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.MonitoringService
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.reports.ImportReportsService
+import java.time.LocalDate
+
+@Component
+class ImportReportsDataTask(
+  @Value("\${IMPORT_REPORTS_BACKDATE_ENABLED}") private val backDateEnabled: Boolean = false,
```

Comment:
This task replaces the two separate tasks for previous days moves, events, journeys task and previous days profiles and peoples task.